docs/datatypes/chain.md

```diff
@@ -1,65 +1,81 @@
-As such it aims to be used where `List` and `Vector` incur a performance penalty.
+[Chain API Documentation][@:api(cats.data.Chain)]
+
+
+`Chain` is an immutable sequence data structure that allows constant time prepending, appending and concatenation.
+This makes it especially efficient when used as a [Monoid], e.g. with [Validated] or [Writer].
+As such it aims to be used where @:api(scala.collection.immutable.List) and @:api(scala.collection.immutable.Vector) incur a performance penalty.
+Cats also includes type class implementations to support using `Chain` as a general-purpose collection type, including [Traverse], [Monad], and [Alternative].
+
+## Motivation
 
 `List` is a great data type, it is very simple and easy to understand.
-It has very low overhead for the most important functions such as `fold` and `map` and also supports prepending a single element in constant time.
+It has very low overhead for the most important functions such as [fold][Foldable] and [map][Functor] and also supports prepending a single element in constant time.
 
-Traversing a data structure with something like `Writer[List[Log], A]` or `ValidatedNel[Error, A]` is  powerful and allows us to precisely specify what kind of iteration we want to do while remaining succint.
+Traversing a data structure with something like [Writer\[List\[Log\], A\]][Writer] or [ValidatedNel\[Error, A\]][Validated] is powerful and allows us to precisely specify what kind of iteration we want to do while remaining succinct.
 However, in terms of efficiency it's a whole different story unfortunately.
-That is because both of these traversals make use of the `List` monoid (or the `NonEmptyList` semigroup), which by the nature of `List` is very inefficient.
-If you use `traverse` with a data structure with `n` elements and `Writer` or `Validated` as the `Applicative` type, you will end up with a runtime of `O(n^2)`.
+That is because both of these traversals make use of the `List` monoid (or the [NonEmptyList] semigroup), which by the nature of `List` is very inefficient.
+If you use [traverse][Traverse] with a data structure with `n` elements and [Writer] or [Validated] as the [Applicative] type, you will end up with a runtime of `O(n^2)`.
 This is because, with `List`, appending a single element requires iterating over the entire data structure and therefore takes linear time.
 
-So `List` isn't all that great for this use case, so let's use `Vector` or `NonEmptyVector` instead, right?
+So @:api(scala.collection.immutable.List) isn't all that great for this use case, so let's use @:api(scala.collection.immutable.Vector) or @:api(cats.data.NonEmptyVector)` instead, right?
 
 Well, `Vector` has its own problems and in this case it's unfortunately not that much faster than `List` at all. You can check [this blog post](http://www.lihaoyi.com/post/BenchmarkingScalaCollections.html#vectors-are-ok) by Li Haoyi for some deeper insight into `Vector`'s issues.
 
```

Comment:
I'm not sure how relevant that blog post is for the 2.13 collections `Vector`. Maybe to 2.12, not sure.

Reply:
It will be interesting to see how relevant Chain remains after 2.13.11 Vector updates.